### ISSUE TITLE
Unable to construct sampler when kinematics not loaded into ompl pipeline

### ISSUE THREAD

**Issue by mikeferguson** [moveit/moveit_ros#324]
The setup assistant no longer generates a line like the following in ompl_planning_pipeline.launch.xml:

```
<rosparam command="load" file="$(find robot_moveit_config)/config/kinematics.yaml"/>
```

This then causes an error when trying to use the pick interface:

```
[ INFO] [1380506826.859073575]: Planning attempt 1 of at most 1
[ INFO] [1380506826.859455926]: Added plan for pipeline 'pick'. Queue is now of size 1
[ERROR] [1380506826.870277203]: No sampler was constructed
[ INFO] [1380506826.870473586]: Manipulation plan 0 failed at stage 'reachable & valid pose filter' on thread 1
[ WARN] [1380506826.870695152]: All supplied grasps failed. Retrying last grasp in verbose mode.
[ INFO] [1380506826.870788668]: Re-added last failed plan for pipeline 'pick'. Queue is now of size 1
[ INFO] [1380506826.879020452]: Manipulation plan 0 failed at stage 'reachable & valid pose filter' on thread 0
[ INFO] [1380506826.879411293]: Pickup planning completed after 0.011495 seconds
```

I presume the plan was to load kinematics from robot_description_kinematics, and that something is missing, but I have no idea where that something is.

**Comment by isucan** [comment]
How are you running your code? planning_context.launch now loads the kinematics information; if you load that, things should work. If not, a block like this should:

```
<group ns="robot_description_kinematics">
    <rosparam command="load" file="$(find [GENERATED_PACKAGE_NAME])/config/kinematics.yaml"/>
</group>
```

**Comment by mikeferguson** [comment]
I used setup_assistant to generate a clean configuration about a week ago. I'm using the move_group.launch, which includes planning_context.launch, which does indeed include robot_description_kinematics. I have no problems calling the move_group action, but the pickup_action fails with the "No sampler was constructed" unless I put the kinematics back into ompl_planning_pipeline.launch.xml. Once  I do that, pick_action works again.

**Comment by mikeferguson** [comment]
fixed by #327
